src/components/calcite-slider/calcite-slider.stories.ts

```diff
@@ -48,6 +48,103 @@ export const Range = (): string => html`
   ></calcite-slider>
 `;
 
+export const RangeLabeledTicks = (): string => html`
+  <calcite-slider
+    min="${number("min", 5)}"
+    min-label="${text("min-label", "Temperature, lower bound")}"
+    min-value="${number("min-value", 95)}"
+    max="${number("max", 100)}"
+    max-label="${text("max-label", "Temperature, upper bound")}"
+    max-value="${number("max-value", 100)}"
+    step="${number("step", 10)}"
+    ${boolean("label-handles", false)}
```

Comment:
I just noticed this after looking at the Screener diffs. Can you also enable the label handles before merging?